### PR TITLE
018_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation

### PR DESCRIPTION
**Pull request by istoimenova**
Used common functions are implemented in
#603
Script is part of TS: #597

### CODE REVIEW

**Comment by istoimenova** [comment]
@OVikhrov, @dboltovskyi, please review.

File: test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+	end
+
+	data.policy_table.app_policies["0000001"] = {
+		keep_context = false,
+		steal_focus = false,
+		priority = "NONE",
+		default_hmi = "NONE",
+		groups = {"Base-4"}
+	}
+	data.policy_table.app_policies["0000001"].AppHMIType = {"NAVIGATION"}
```
Comment (aderiabin) [inline test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua]: @istoimenova 
I propose move AppHMIType parameter initialization to structure above:
```
	data.policy_table.app_policies["0000001"] = {
 		keep_context = false,
 		steal_focus = false,
 		priority = "NONE",
 		default_hmi = "NONE",
 		groups = {"Base-4"},
		AppHMIType = {"NAVIGATION"}
 	}
```

File: test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+	EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = config.application1.registerAppInterfaceParams.appName }})
+	EXPECT_RESPONSE(CorIdRegister, { success = true, resultCode = "WARNINGS", hmiCapabilities = { steeringWheelLocation = value_steering_wheel_location } })
+	EXPECT_NOTIFICATION("OnHMIStatus", { systemContext = "MAIN", hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE"})
+end
+
+--[[ Postconditions ]]
+commonFunctions:newTestCasesGroup("Postconditions")
+
+function Test.Postcondition_Restore_config_files()
+	commonPreconditions:RestoreFile("hmi_capabilities.json")
```
Comment (aderiabin) [inline test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua]: @istoimenova 
I propose remove call of function `commonPreconditions:RestoreFile("hmi_capabilities.json")` take into account that hmi_capabilities.json was not stored in this test script.

File: test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua (hunk 1 above)
Comment (istoimenova) [inline test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua]: @aderiabin, done in bdc5e79ac756997f237d852b80555988e00a9ea6

File: test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua (hunk 2 above)
Comment (istoimenova) [inline test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua]: @aderiabin, done in bdc5e79ac756997f237d852b80555988e00a9ea6

Comment (aderiabin) [inline test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua]: @istoimenova 
You have remove call of function 
`commonPreconditions:RestoreFile("sdl_preloaded_pt.json").`
Please get it back, it is used necessarily and remove function `commonPreconditions:RestoreFile("hmi_capabilities.json")` instead.

Comment (istoimenova) [inline test_scripts/API/RAI/046_ATF_N_HMI_Does_Not_Respond_WARNINGS_RAI_steeringWheelLocation.lua]: @aderiabin, thank you. done in 2d2eceec12b4fbe3da2fc998fea9c7a10261556e

**Comment by istoimenova** [comment]
@OVikhrov, @aderiabin, see updates in 2d2ecee

**Pull request merged**